src/Command/BackupCommand.php

```diff
@@ -26,15 +26,18 @@ class BackupCommand extends Command
     private ?array $excludedTables = null;
     private ?string $extraOptions = null;
     private bool $ignoreDefaultOptions = false;
+    private float $timeout;
 
     public function __construct(
         string $defaultConnectionName,
         private BackupperFactory $backupperFactory,
         private Storage $storage,
+        float $timeout,
     ) {
         parent::__construct();
 
         $this->connectionName = $defaultConnectionName;
+        $this->timeout = $timeout;
```

Comment:
```suggestion=
    public function __construct(
        string $defaultConnectionName,
        private BackupperFactory $backupperFactory,
        private Storage $storage,
        private float $timeout,
    ) {
        parent::__construct();

        $this->connectionName = $defaultConnectionName;
```

Reply:
We try to specify private properties in constructor to avoid boilerplate code as much as possible :)

Reply:
I see, should i also change it for `$this->connectionName = $defaultConnectionName;`? since that is not my code, that was already there.

Or was this done on purpose because its important that the command has the parameter `defaultConnectionName`, but the property is named `connectionName `?

Reply:
I think the `$defaultConnectionName` might be wrong in this context, I'm OK with naming the parameter `$connectionName`.